tasks/copy_scripts.yml

```diff
@@ -43,20 +43,20 @@
   template:
     src: 'notify_telegram.sh.j2'
     dest: "{{ mirror_scripts_path }}/notify_telegram.sh"
-    owner: root
+    owner: "{{ mirror_user.user }}"
     mode: 0766
 
 - name: Page init
   template:
     src: index.html.j2
-    dest: /var/www/html/index.html
-    owner: root
+    dest: /var/www/html/mirror/index.html
```

Comment:
This path need to be parameterized, don't you think?

Reply:
Parameterized var mirror_page_init